File: fix_root.py
import re

with open("gitmap/cmd/root.go", "r", encoding="utf-8") as f:
    content = f.read()

# Replace: if command == "agy" {
content = content.replace('if command == "agy" {', 'if command == "agy" || command == "ag" || command == "antigravity" {')

with open("gitmap/cmd/root.go", "w", encoding="utf-8") as f:
    f.write(content)

print("done")

File: fix_root.go
package main

import (
	"os"
	"strings"
)

func main() {
	b, _ := os.ReadFile("gitmap/cmd/root.go")
	s := string(b)
	s = strings.ReplaceAll(s, "cliexit.Reportf(command, \"execute\", \"\", err)\n\t\t\tpanic(\"fatal error\")", "handleGlobalError(command, err)")
	s = strings.ReplaceAll(s, "cliexit.Reportf(command, \"execute\", \"\", err)\r\n\t\t\tpanic(\"fatal error\")", "handleGlobalError(command, err)")

	if !strings.Contains(s, "\"github.com/alimtvnetwork/gitmap-v28/gitmap/apperror\"") {
		s = strings.Replace(s, "\"github.com/alimtvnetwork/gitmap-v28/gitmap/cliexit\"", "\"github.com/alimtvnetwork/gitmap-v28/gitmap/apperror\"\n\t\"github.com/alimtvnetwork/gitmap-v28/gitmap/cliexit\"\n\t\"github.com/alimtvnetwork/gitmap-v28/gitmap/config\"", 1)
	}

	os.WriteFile("gitmap/cmd/root.go", []byte(s), 0644)
}
